format_implementing.md

```diff
@@ -104,4 +98,38 @@ Another condition you may see often is `required`
 ]
 ```
 
-If your program considers itself a client, the mod is required. If your program considers itself a server, the mod is not required to be installed.
\ No newline at end of file
+If your program considers itself a client, the mod is required. If your program considers itself a server, the mod is not required to be installed.
+
+---
+
+## Implementing `game`
+When parsing a version of a project, the `game` field specifies which Minecraft version it's compatible with.
+
+Unfortunately, Minecraft doesn't follow SemVer, so it's not very easy to list supported versions. Fortunately, the current system of the `game` field provides the ability to list supported versions in the majority of cases.
+
+### `minimum`
+This field should be taken strictly. Any version before this version, the project will simply not function at all.
+
+```json
+"game": {
+  "minimum": "1.16.3"
+}
+```
+In the above example, the project will not work at all on any version prior to 1.16.3, including 1.16.2 or 1.16.1
+
+### `maximum`
+This field should not be taken as strict as `minimum`. Authors can't know what the future holds. It's recommended for authors to put the next major version (e.g. for a 1.16.2 mod, put `1.17`) in this field, if they decide to have the `maximum` field.
+
+For launchers, there are two routes you can go - either refuse to install mods for which the maximum version check fails, or warn the user about the problem. 
+
+### `breaks`
+These are versions where it's known that the mod doesn't work. You can take this strictly.
+
+### FAQ
+**Q: Where do I get Minecraft version metadata, if it's not specified anywhere?**  
+A: Most launchers will already have this. If not, the recommended place is Mojang's official metadata, available at http://launchermeta.mojang.com/mc/game/version_manifest.json
```

Comment:
All launchers get it here. It's not a case of "you probably have the data, if not then get it here" - it's simply "get it here"